data_structures/queue_with_stacks.py

```diff
@@ -0,0 +1,34 @@
+from stack.stack import Stack
+
+
+class Queue_with_stacks:
+
+    def __init__(self, iterable=[]):
+        self.stack_back = Stack(iterable)
+        self.stack_front = Stack()
+
+    def __topval__(self):
+        if self.stack_back.top.val is None:
+            print('Queue is empty')
+        else:
+            print('Next in queue is {}'.format(self.stack_back.top.val))         
+
+    def enqueue(self, val):
+        self.stack_back.push(val)
+        print('{} added to end of Queue'.format(self.stack_back.top.val))
+
+    def dequeue(self):
+        if self.stack_back.top is None:
+            raise IndexError("queue is empty")
+        x = None
+        while self.stack_back.top._next:
```

Comment:
You are supposed to be using the stack as a black box. That is, you cannot access these specific properties on the stack, only the `.push()` and `.pop()` methods.